File: common/protcol/c#/module/centermodule.cs
/*this module file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.Collections.Generic;

namespace module
{
    public class center : juggle.Imodule 
    {
        public center()
        {
			module_name = "center";
        }

        public delegate void reg_serverhandle(String argv0, String argv1, Int64 argv2, String argv3, Int64 argv4);
        public event reg_serverhandle onreg_server;
        public void reg_server(ArrayList _event)
        {
            if(onreg_server != null)
            {
                var argv0 = ((String)_event[0]);
                var argv1 = ((String)_event[1]);
                var argv2 = ((Int64)_event[2]);
                var argv3 = ((String)_event[3]);
                var argv4 = ((Int64)_event[4]);
                onreg_server( argv0,  argv1,  argv2,  argv3,  argv4);
            }
        }

	}
}

File: common/protcol/js/module/centermodule.js
/*this module file is codegen by juggle for js*/
function center_module(){
    eventobj.call(this);
    Imodule.call(this, "center");

    this.reg_server = function(argv0, argv1, argv2, argv3){
        this.call_event("reg_server", [argv0, argv1, argv2, argv3]);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Imodule.prototype;
    center_module.prototype = new Super();
})();
center_module.prototype.constructor = center_module;

